### PR TITLE
Generate api reference document

### PR DESCRIPTION
**Pull request by zwtop**
Signed-off-by: zwtop <wang.zhan@smartx.com>

### CODE REVIEW

**Comment by zwtop** [comment]
We can use [netlify](https://netlify.com) to hosting ower website.
e.g. https://everoute.netlify.app

It is also the choice for some open-source projects:
https://kubernetes.netlify.app (https://kubernetes.io)
https://etcd.netlify.app (https://etcd.io)
https://cert-manager.app (https://cert-manager.io)

Netlify open-source policy: https://www.netlify.com/legal/open-source-policy

File: Makefile [hunk 1]
```diff
@@ -59,6 +59,12 @@ gqlgen:
 protopb:
 	protoc -I=. --go_out=plugins=grpc:.  pkg/apis/cni/v1alpha1/cni.proto
 
+apidocs-gen:
+	go install github.com/ahmetb/gen-crd-api-reference-docs@v0.3.0
```
Comment (wu-cl) [inline Makefile:63]: maybe there is no need to install it each time we generate code.

**Comment by KeyongSun** [comment]
企业微信截图_c0434c43-ae8f-4cbd-a554-e09885053eda

File: Makefile (hunk 1 above)
Comment (zwtop) [reply Makefile:63]: maybe we should check it before install.

**Review by KeyongSun — APPROVED** [review]

**Pull request merged**